File: chuno/src/main/java/com/leesfamily/chuno/room/model/DateTime.java
package com.leesfamily.chuno.room.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Embeddable;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Embeddable
public class DateTime {
    private int year;
    private int month;
    private int day;
    private int hour;
    private int minute;
}

File: Chuno/app/src/main/java/com/leesfamily/chuno/network/data/DateTime.kt
package com.leesfamily.chuno.network.data

data class DateTime(
     var year: Int,
     var month: Int,
     var day: Int,
     var hour: Int,
     var minute: Int,

) {
     fun dateToString(): String {
          return "$year-$month-$day"
     }
}
